hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -145,13 +145,13 @@
     {
       name: roll_cg
       desc: '''
-            Covers that all valid enable and polarity settings
-            for the Pattgen control register have been tested.
-            Individual enable and polarity settings that will be covered include:
-            - Enable pattern generator functionality for Channel 0 (ENABLE_CH0)
-            - Enable pattern generator functionality for Channel 1 (ENABLE_CH1)
-            - Clock (pcl) polarity for Channel 0 (POLARITY_CH0)
-            - Clock (pcl) polarity for Channel 1 (POLARITY_CH1)
+            Covers that all counters revert to zero when they reach
+            the maximum value.
+            Individual initial and maximum counter values
+            that will be covered include:
```

Comment:
it is just one counter or several?
you should add a "the" or "counters" or change "clock divide counter" 
to the actual clock name.
something that will make it more clear

Reply:
Corrected description